src/actions.js

```diff
@@ -5562,6 +5562,19 @@ export function setAction(c_action,action,type,old,prediction){
             },
             repairMax(){
                 return c_action.repair();
+            },
+            mouse_enter () {
+                const title = this.title;
+                $('.buildList .queued span[class^="res-"]').each(function () {
+                    if (this.innerText === title) {
```

Comment:
If you wanted to have this `if` to be part of the selector, I found `[id^=q${id}] span:first-child` could replace `.queued span[class^="res-"]`